### DISCUSSION TITLE
添加正版披风预览

### DISCUSSION THREAD

**Discussion by Crotina** [Meloong-Git/PCL#4792] [提问] [answered]
在pcl给正版账户换披风的时候，为披风列表增加简单的预览

**Comment by 3gf8jv4dv** [comment]
You can open an issue and I will let LTCatt decide.

**Comment by 3gf8jv4dv** [comment]
Umm... Looks like it was deleted by the bot due to being posted too frequently.

Let @LTCatt take a look at this.

**Comment by LTCatt** [comment, marked as answer]
这个作用不是很大，暂时不考虑，抱歉（
